src/components/general/Collapser.js

```diff
@@ -34,7 +32,7 @@ function Collapser({
     externalCollapsed,
     icon,
 }: Props): ReactElement<"div"> {
-    const [isCollapsed, setIsCollapsed] = useState(!initiallyExpanded);
+    const [isCollapsed, setIsCollapsed] = useState(true);
```

Comment:
Looks good, you will need to change all places that use `initiallyExpanded` as a prop to `externalCollapsed`